lib/js/components/SectionHeader/SectionHeader.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+	<div class="sectionHeader" :class="{ '-expandable': expandable, [sizeClass]: true }">
+		<div class="sectionHeader__textWrapper" @click="onTextWrapperClicked">
+			<ds-icon
+				v-if="iconLeft"
+				class="sectionHeader__iconLeft"
+				:icon="iconLeft"
+				:size="ICON_SIZES.SMALL"
+			/>
+			<span class="sectionHeader__text">{{ text }}</span>
+			<ds-icon
+				v-if="expandable"
+				class="sectionHeader__chevron"
+				:icon="ICONS.FA_CHEVRON_DOWN"
+				:rotation="chevronRotation"
+				:size="ICON_SIZES.SMALL"
+			/>
+		</div>
+		<slot></slot>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/media-queries';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.sectionHeader {
+	$section-header-height-large: 56px;
+	$section-header-height-medium: 44px;
+	$self: &;
+
+	align-items: center;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+
+	@media #{breakpoint-s()} {
+		flex-wrap: nowrap;
+	}
+
+	&.-size-l {
+		min-height: 56px;
```

Comment:
In Figma, I see `padding` instead of `min-height`

Reply:
Padding is set on icon wrappers and you can have a header without icons. I believe the height shouldn't depend on the icons presence. Same discussion in another context: https://www.figma.com/file/HFY6waq7kWsr3rj8a0ghXZ?node-id=142:68001#187503550

Reply:
makes sense :+1:

Reply:
On the other hand, if you don't add padding on text, then it'll look strange when the text goes into second line without anny padding